src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -23,6 +30,16 @@ export default class AccountDeposit extends Component {
       height: 'auto',
       width: '100%',
     }
+    const shapeShiftButton = {
```

Comment:
Please move to less style sheet.

Reply:
Sorry, I'm pretty new to the UI code. Which CSS file should this be moved to?

Reply:
Never mind, looks like it's `account-deposit.styles.less`. I'll add it there.

Reply:
Awesome!